streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1431,6 +1434,10 @@ private Thread shutdownHelper(final boolean error) {
     }
 
     private boolean close(final long timeoutMs) {
```

Comment:
Do we still need this helper? Seems it's called only in a single place?

Reply:
Good catch.  Removing it.